### ISSUE TITLE
Using with Leaflet

### ISSUE THREAD

**Issue by daniyel** [klokantech/tileserver-gl#58]
Hi.

We are using the tileserver-gl for our new maps. We are already using Leaflet.js for interactions with the map. So far the tileserver-gl viewer is working, we can navigate through the map, we see our styles, font family, etc. But when we load the URL through the Leaflet then we are getting 404 errors. Are we doing something wrong? For maps we are using planet.mbtiles, which we downloaded recently from http://osm2vectortiles.org/ page.

URL we are using is: **https://subdomain.domain.net/styles/js/{z}/{x}/{y}.png**

And Leaflet code goes:

```
...
var map = L.map(id);

L.tileLayer('https://subdomain.domain.net/styles/js/{z}/{x}/{y}.png', {
	maxZoom: 18
}).addTo(map);

map.scrollWheelZoom.disable();
...
```

config.json
```
{
  "options": {
    "paths": {
      "root": "",
      "fonts": "glyphs",
      "sprites": "sprites",
      "styles": "styles",
      "mbtiles": "data"
    },
    "formatEncoding": {
      "png": 6,
      "jpeg": 80,
      "webp": 90
    }
  },
  "styles": {
	"js": {
      "style": "js.json",
      "tilejson": {
        "type": "overlay",
        "bounds": [8.529446, 47.364758, 8.55232, 47.380539]
      }
    }
  },
  "data": {
    "planet-vector": {
      "mbtiles": "planet.mbtiles"
    }
  }
}
```
in styles/js.json we have this part as:
```
...
"sources": {
   "mapbox": {
        "url": "mbtiles://planet.mbtiles",
        "type": "vector"
    }
},
...
```

Are we missing something?

Best regards.

**Comment by petrsloup** [comment]
Hi,

if you try to access some tiles manually (e.g. https://subdomain.domain.net/styles/js/0/0/0.png) is it working correctly?

**Comment by daniyel** [comment]
I always get: Cannot GET /styles/js/13/4094/2723.png

**Comment by petrsloup** [comment]
I just realized the URL is incorrect. Correct URL to use: https://subdomain.domain.net/styles/js/rendered/{z}/{x}/{y}.png (note the `rendered/` part in path).

This is a bug in the index.html of tileserver-gl where the URL is displayed incorrectly.

Going to fix it ASAP

**Comment by daniyel** [comment]
Yes with rendered in URL it works :)

Full URL example: https://subdomain.domain.net/styles/js/rendered/13/4092/2724.png

**Comment by petrsloup** [comment]
Fixed in v1.1.4
